app/src/main/java/com/asfoundation/wallet/App.java

```diff
@@ -104,17 +107,26 @@ private void initializeRakam() {
       Log.e(TAG, "error: ", e);
     }
 
+    int userLevel = idsRepository.getGamificationLevel();
+    String installerPackage =
```

Comment:
can we use the idRepository to get the installer? In the case the logic to obtain the installer is updated. We already have the get installer logic in another service. Are we able to reuse it?